nectar/src/config/settings.rs

```diff
@@ -408,6 +414,10 @@ impl From<Settings> for File {
             }),
             bitcoin: Some(bitcoin.into()),
             ethereum: Some(ethereum.into()),
+            sentry: match sentry {
+                None => None,
+                Some(sentry) => Some(file::Sentry { url: sentry.url }),
+            },
```

Comment:
Please follow how the existing code config code is handled here and extract the actual mapping into a function of some sorts.

You could also utilize `Option::map` here.

Reply:
My brain does not work perfectly atm, does the last patch fix this?